### PR TITLE
stig: fix build

### PR DESCRIPTION
**Pull request by ju1m**
###### Motivation for this change
Fix build.

###### Things done
- [X] Disabling failing test.
- [X] Upstream issue: https://github.com/rndusr/stig/issues/214

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 147806` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>stig</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
Change looks good to me.

Fix build (https://hydra.nixos.org/build/159546950)

File: pkgs/applications/networking/p2p/stig/default.nix [hunk 1]
```diff
@@ -44,6 +44,8 @@ python3Packages.buildPythonApplication rec {
     "--deselect=tests/client_test/ttypes_test.py::TestTimestamp::test_string__month_day_hour_minute_second"
```
Comment (doronbehar) [inline pkgs/applications/networking/p2p/stig/default.nix:44]: Could you check while at it if the other tests disabled should stay disabled? Please. For example there's a test disabled which a comment above it indicates it fails on Darwin, but it's disabled for linux as well.

Comment (ju1m) [reply pkgs/applications/networking/p2p/stig/default.nix:44]: Test works for me on Linux, I've moved it to the Darwin list: https://github.com/NixOS/nixpkgs/pull/147833

**Pull request merged**